src/styles.js

```diff
@@ -1623,7 +1631,7 @@ export const schoolInformationStyles = StyleSheet.create({
 
 	error: {
 		fontFamily: 'Raleway-Regular',
-		color: red,
+		color: 'red',
```

Comment:
Maybe create a const inside the styles.js eg  error-red = 'red'; and then import that rather than hardcoding it